docs/UserGuide.md

```diff
@@ -23,22 +23,22 @@ Managing clients and a busy schedule has never been easier. With OutBook, all yo
 - [Features](#features)
   - [Contact Commands](#contact-commands)
     - [Contact Parameters](#contact-parameters)
-    - [Adding a contact: `addc`](#adding-a-contact-addc)
-    - [Listing all contacts: `listc`](#listing-all-contacts-listc)
-    - [Deleting a contact: `deletec`](#deleting-a-contact-deletec)
+    - [Adding a contact: `addc`](#adding-a-contact--addc)
```

Comment:
the double hyphen doesn't work on the site even though it does on intellij. it was like this in a previous commit but i had to change it because the links didn't work on the generated HTML